patch/atmega328p.yaml

```diff
@@ -3,6 +3,7 @@ _svd: ../svd/atmega328p.svd
 _include:
   - "common/ac.yaml"
   - "common/adc.yaml"
+  - "common/cpu.yaml"
```

Comment:
I do not think this is a common patch, the `PRR` register(s) seem to differ wildly between the different MCUs.  Thus I would suggest moving the patch directly into this file (below the includes).